### PR TITLE
Standardize comments and other formatting issues

### PR DESCRIPTION
**Pull request by kevinchan159**
## Overview
Fix formatting issues with comments, indentation, spaces, and other minor issues.

## Changes Made

- Fixed comments to distinguish between `///` and `//`. `//` should only be used in function comments.
- Removed all instances of `/** /`
- Fixed indentation for lines that are too long
- Add/remove newlines
- Fixed `// MARK` comments

## Test Coverage

Ran the app

### CODE REVIEW

**Review by haiyingweng — APPROVED** [review]
Wow! :clap:

File: TCAT/AppDelegate.swift [hunk 1]
```diff
@@ -151,7 +151,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // Called when the application is about to terminate. Save data if appropriate. See also applicationDidEnterBackground:.
     }
     
-    // MARK: Helper Functions
+    // MARK: - Helper Functions
```
Comment (haiyingweng) [inline TCAT/AppDelegate.swift:154]: Question: 
When is `MARK:` used usually?

Comment (kevinchan159) [reply TCAT/AppDelegate.swift:154]: its usually used to denote a section in a file. when you add `// MARK: -` in a file, XCode will allow you to directly jump to that section quickly in the IDE.

File: TCAT/Views/SummaryView.swift [hunk 2]
```diff
@@ -27,13 +27,11 @@ class SummaryView: UIView {
...
-         This value ends up getting overwritten by constraints, which is what we want,
-         but for some reason if it is not set prior to writing the constraints, the
-         entire view comes out blank. I'm still investigating but it seems to be an,
-         issue with the Pulley Pod that we're using.
-        */
+        // TODO:
+        // This value ends up getting overwritten by constraints, which is what we want,
+        // but for some reason if it is not set prior to writing the constraints, the
+        // entire view comes out blank. I'm still investigating but it seems to be an,
+        // issue with the Pulley Pod that we're using.
```
Comment (lucyxubroad) [inline TCAT/Views/SummaryView.swift:34]: What is the reason that we don't want to use /**/ for multiline comments?

Comment (kevinchan159) [reply TCAT/Views/SummaryView.swift:34]: i'm following Google's style guidelines and `/** */` is used for Java docs so we shouldn't use them for Swift

**Pull request merged**